### ISSUE TITLE
How does the .Preprocessor in ObjectList work?

### ISSUE THREAD

**Issue by Leandros** [fastbuild/fastbuild#219]
How does the `.Preprocessor` & `.PreprocessorOptions` options of `ObjectList` work?
Adding `-verbose` shows me the preprocessor is getting invoked:

```
[...]
3> -Build: 0 ms C:\Users\leandros\workspace\scratch\jit\pre.exe
[...]
```

but it doesn't seem to be actually calling the executable, since I neither get any output in `stdout` or `stderr` printed out, nor does it create files I was creating (this was added as a test, to see if fastbuild is actually calling the executable).

How is the preprocessor supposed to work? What is he required to do, how can I get the current file as an input to the preprocessor, where does the output has to go? Does it replace the C / C++ preprocessor or act separately? Does it act before or after?

**Comment by nre-ableton** [comment]
@Leandros see #221, it seems that `.PreprocessorOptions` is broken in 0.93. You should try using a build of version 0.92 in the meantime.

**Comment by ffulin** [comment]
I think @nre-ableton is right. More details on the linked issue.

**Comment by ffulin** [comment]
I'm going to close this issue as a duplicate of https://github.com/fastbuild/fastbuild/issues/221 - that bug has been resolved and will be fixed in the next version (v0.94). As a work-around, you can use v0.92 until that is avaiable.
